design/one-pager-managed-resource-api-design.md

```diff
@@ -0,0 +1,265 @@
+# Managed Resources API Patterns
+* Owner: Muvaffak Onus (@muvaf)
+* Reviewers: Crossplane Maintainers
+* Status: Draft
+
+## Terminology
```

Comment:
Should we link to the existing glossary?
https://github.com/crossplaneio/crossplane/blob/master/docs/concepts.md#glossary

Reply:
I think terminology section provides information about just the terms that you need to know for that document, so, it's more contained and easy to digest than full glossary. But adding the glossary link in the end of that section makes sense probably.